### PR TITLE
test(init): add tests for the `init` command

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -2,17 +2,18 @@ name: CI
 on: [pull_request]
 
 jobs:
-    build:
+    ci:
```
Comment (Nevoss) [inline .github/workflows/ci.yml:5]: Can you move it to another PR?

File: jest.config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
```
Comment (Nevoss) [inline jest.config.js:1]: Maybe we should try `vitest`? and then there is not need for extra packages or configuration?

File: src/create-program.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import { Command } from 'commander';
+import { packageName } from './consts';
+import { install, init } from './commands';
+import wrapCommand from './utils/wrap-command';
+
+export function createProgram() {
```
Comment (Nevoss) [inline src/create-program.ts:6]: Can you move this code to the index file?

**Review by Nevoss — CHANGES_REQUESTED** [review]
Cool! Thanks! 😄

File: .github/workflows/ci.yml (hunk 1 above)
Comment (StyleShit) [reply .github/workflows/ci.yml:5]: reverted. will add in another PR after this one

File: jest.config.js (hunk 2 above)
Comment (StyleShit) [reply jest.config.js:1]: can't make `vi.mock` to work on recursive imports (`fs` for `fs-extra`) like `jest.mock`

File: src/create-program.ts (hunk 3 above)
Comment (StyleShit) [reply src/create-program.ts:6]: can't, it'll run the `main()` function in tests

**Review by Nevoss — APPROVED** [review]
